File: src/main/java/io/github/incplusplus/bigtoolbox/network/wlan/interop/MessageBool.java
package io.github.incplusplus.bigtoolbox.network.wlan.interop;

public class MessageBool implements java.io.Serializable
{
	public String message;
	public boolean success;

	public MessageBool(boolean success, String message)
	{
		this.success = success;
		this.message = message;
	}

	public boolean successful()
	{
		return success;
	}

	public String getMessage()
	{
		return message;
	}
}

File: src/main/dotnet/WinWiFi/SimpleWifi/MessageBool.cs
namespace SimpleWifi
{
    public class MessageBool
    {
        public string message;
        public bool success;

        public MessageBool(bool success, string message)
        {
            this.success = success;
            this.message = message;
        }

        public bool successful()
        {
            return success;
        }

        public string getMessage()
        {
            return message;
        }
    }
}